### PR TITLE
Fix after NIJZ change queries

### PR DESCRIPTION
**Pull request by jalezi**
It's a DRAFT.

Crurrently used by [data/update_vaccination.py](https://github.com/sledilnik/data/blob/master/update_vaccination.py):
* [x] vaccinations_by_day()
* [x] vaccines_supplied_by_manufacturer()
* [ ] vaccinations_by_manufacturer_supplied_used()
* [x] vaccinations_by_age()
* [x] vaccinations_by_region_by_day()
* [x] vaccinations_by_municipalities_share()

Set up a test PR https://github.com/sledilnik/data/pull/132 to test the changes: [Vaccination update](https://github.com/sledilnik/data/actions/workflows/vaccination.yml?query=branch%3Acepimose-fix-after-nijz-change)

### CODE REVIEW

File: cepimose/data.py [hunk 1]
```diff
@@ -1463,9 +1471,40 @@ def _create_vaccination_by_manufacturer_used_requests():
...
+                        "Expression": {
+                            "Column": {
+                                "Expression": {"SourceRef": {"Source": "e"}},
+                                "Property": "Polno ﻿Cepljeni",
+                            }
+                        },
+                        "Function": 0,
                     },
-                    "Name": "eRCO_podatki_obcine_pop.Odst_Obcina_2",
+                    "Name": "Sum(eRCO_podatki_občine.PolnöCepljeni)",
```
Comment (stefanb) [inline cepimose/data.py:1512]: typo? Poln**ö**Cepljeni -> Poln**o**Cepljeni

Comment (jalezi) [reply cepimose/data.py:1510]: verjetno  je typo pri njih

Comment (jalezi) [reply cepimose/data.py:1510]: verjetno  je typo pri njih

Comment (stefanb) [reply cepimose/data.py:1510]: zänïmïvö 😄

File: cepimose/parser.py [hunk 2]
```diff
@@ -49,15 +67,22 @@ def _parse_vaccinations_by_day(data) -> "list[VaccinationByDayRow]":
...
-        count_first = int(element["X"][0]["C"][1])
-        count_second = int(element["X"][1]["C"][1])
-        share_first = float(element["X"][0]["C"][0]) / 100.0
-        share_second = float(element["X"][1]["C"][0]) / 100.0
+        C = element["C"]
+        age_group = str(C[0])
+        share_second = float(C[1]) / 100.0
+        share_first = float(C[2]) / 100.0
+        count_second = int(C[3])
+        count_first = int(C[4])
```
Comment (stefanb) [inline cepimose/parser.py:85]:
```suggestion
        count_first = int(C[3])
        count_second = int(C[4])
```
but check also the shares above them

Comment (stefanb) [reply cepimose/parser.py]: fixed in 2ba3984

**Pull request merged**